src/systems/menu_defn.cpp

```diff
@@ -1516,7 +1513,7 @@ static const char* JUMP_COUNTS[] = {"One", "Two", "Infinite"};
 static Widget s_jump_classic_widgets[] = {
     {
         .type = WidgetType::Text,
-        .text = {"  Classic Jump-Mod from it's original release"},
+        .text = {"  Classic Jump-Mod from its original release"},
```

Comment:
I see Jump-Mod in some places and Jump Mod in others, could you make these consistent or justify the inconsistency>